pyproject.toml

```diff
@@ -92,4 +93,7 @@ build = [
 
 [tool.fixit]
 enable = ["fixit.rules"]
-disable = ["fixit.test.foo", "fixit.test.bar"]
+disable = [
+    "fixit.rules:ExplicitFrozenDataclassRule",
+    "fixit.rules:UseLintFixmeCommentRule",
```

Comment:
maybe add a comment explaining the reason for disabling these? Feels weird to ship a lint rule as part of fixit but not enable it on fixit itself :)

Reply:
Sure thing. These feel like *very* opinionated rules, and I would like to have a discussion about moving them (and maybe others?) into an "extras" subpackage that aren't selected by default (with the new collection behavior), or maybe extracting them into a separate package/plugin altogether.